docs/csharp/whats-new/csharp-7.md

```diff
@@ -108,9 +108,8 @@ names to each of the members of the tuple:
 [!code-csharp[NamedTuple](../../../samples/snippets/csharp/new-in-7/program.cs#05_NamedTuple "Named tuple")]
 
 > [!NOTE]
-> The new tuples features require the `System.ValueTuple` type. For Visual Studio 15
-> Preview 5 and earlier preview releases, you must add the NuGet package "System.ValueTuple",
-> available in the pre-release stream.
+> The new tuples features require the `System.ValueTuple` type.
```

Comment:
Idk ... but this might render on two lines. Would it be safer to set it up as:

> The new tuples features require the `System.ValueTuple` type. You must add the NuGet package `System.ValueTuple` in order to use it.

Also, how "new" is "new" when hardening the doc for the future? How do you feel about:

> The tuples features require the `System.ValueTuple` type, and you must add the NuGet package `System.ValueTuple` in order to use it.